php/utils.php

```diff
@@ -447,19 +447,6 @@ function run_mysql_command( $cmd, $assoc_args, $descriptors = null ) {
  * IMPORTANT: Automatic HTML escaping is disabled!
  */
 function mustache_render( $template_name, $data = array() ) {
-	// Transform absolute path to relative path inside of Phar
-	if ( inside_phar() && 0 === stripos( $template_name, PHAR_STREAM_PREFIX ) ) {
-		$search = '';
-		$replace = '';
-		if ( file_exists( WP_CLI_ROOT . '/vendor/autoload.php' ) ) {
-			$search = dirname( __DIR__ );
-			$replace = WP_CLI_ROOT;
-		} elseif ( file_exists( dirname( dirname( WP_CLI_ROOT ) ) . '/autoload.php' ) ) {
-			$search = dirname( dirname( dirname( __DIR__ ) ) );
-			$replace = dirname( dirname( dirname( WP_CLI_ROOT ) ) );
-		}
-		$template_name = str_replace( $search, $replace, $template_name );
-	}
```

Comment:
This was transforming the `$template_name` to make its path work from within the Phar. It is not needed anymore as templates are passed through `phar_safe_path()`, and besides that it was broken under some circumstances.